addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -477,6 +477,15 @@ options.Class.include({
                 return weUtils.getCSSVariableValue(params.variable);
             }
             case 'customizeWebsiteColor': {
+                if (params.variable) {
+                    let variableValue = weUtils.getCSSVariableValue(params.variable);
+                    if (variableValue) {
+                        if (variableValue.startsWith("'") && variableValue.endsWith("'")) {
+                            variableValue = variableValue.substring(1, variableValue.length - 1);
+                        }
+                        return variableValue;
+                    }
+                }
```

Comment:
Then it would pollute the non-color-related variable side instead.
I changed the parameter names now so that the gradient variable name is "data-color-gradient" (instead of the confusing "data-variable" recycled from the other method).